packages/ui-components/src/components/Matrix/MatrixRow.tsx

```diff
@@ -139,6 +153,7 @@ export const MatrixRow = ({ row, parents = [] }: MatrixRowProps) => {
             rowTitle={row.title}
             colKey={key}
             isCategory={isCategory}
+            link={link}
```

Comment:
Sorry one more comment. I think this makes every Cell a clickable link, right? Only the first column should be clickable.